application/forms/ChannelForm.php

```diff
@@ -41,114 +57,27 @@ protected function assemble()
             ]
         );
 
-        $type = [
-            ''            => sprintf(' - %s - ', $this->translate('Please choose')),
-            'email'       => $this->translate('Email'),
-            'rocketchat'  => 'Rocket.Chat'
-        ];
-
         $this->addElement(
             'select',
             'type',
             [
                 'label'    => $this->translate('Type'),
                 'class'    => 'autosubmit',
                 'required' => true,
-                'disable'  => [''],
-                'options'  => $type
+                'disabledOptions'  => [''],
```

Comment:
Either indent all `=>` to the same level, or all with a single space before and after.